src/renderer/utils/readableStream.ts

```diff
@@ -21,6 +21,18 @@
 
 import { Readable } from "readable-stream";
 
+type TypeArray = Int8Array
+  | Uint8Array
+  | Uint8ClampedArray
+  | Int16Array
+  | Uint16Array
+  | Int32Array
+  | Uint32Array
+  | Float32Array
+  | Float64Array
+  | BigInt64Array
+  | BigUint64Array;
+
```

Comment:
We have `type-fest` already in the project where this available as `TypedArray`.

Reply:
Oh cool, I was slightly annoyed that it wasn't part of the default typings. I think we can probably merge this and then change it in a follow up PR.